examples/p5js/KMeans/KMeans_mouseClustering/sketch.js

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I think there will be some lint errors (for example, adding spaces for brackets in function definitions(, but I'm in the process rethinking the linting strategy. Let's merge this in as-is and I may return to to make changes to fix the lint errors!